src/routes/feedback/components/FeedbackModal.js

```diff
@@ -54,7 +54,7 @@ class FeedbackModal extends React.Component<PropsType, StateType> {
 
   onFeedbackCommentChanged = (comment: string) => this.setState({ comment })
 
-  onSubmit = () => {
+  onSubmit = async () => {
```

Comment:
This `async` is not needed here, as the promises are handled using `then` and `catch` below.
```suggestion
  onSubmit = () => {
```